src/main/resources/interface-specification.yml

```diff
@@ -80,6 +80,10 @@ paths:
               $ref: '#/components/schemas/PurposeVersionState'
             default: []
           explode: false
+        - in: query
+          name: excludeDraft
+          schema:
+            type: boolean
```

Comment:
You could add a `default: false` to remove the optionality in the code.
This way it is also clear at spec level which is the default behaviour

Reply:
+1